plugins/zenoh-plugin-rest/examples/z_serve_sse.rs

```diff
@@ -15,6 +15,7 @@ use clap::{arg, Command};
 use std::time::Duration;
 use zenoh::prelude::r#async::*;
 use zenoh::publication::CongestionControl;
+use zenoh::sample_builder::QoSBuilderTrait;
```

Comment:
Similar. Trait should be part of the prelude.

Reply:
done